### PR TITLE
cv design

### PR DESCRIPTION
**Pull request by Ziaurrahaman7**

### CODE REVIEW

**Comment by mazharul** [comment]
Hey - Add in a redirect rule for netlify redirect to redirect the content you have here to `domain.com/cv`

Here is a detail of how to do it: https://docs.netlify.com/routing/redirects/

**Comment by mazharul** [comment]
Why did you edit the current index page? Commit your code to a new page. Name it `cv.html`

**Comment by mazharul** [comment]
Don't change existing files.

**Comment by Ziaurrahaman7** [comment]
can i push index.html file rename with cv.html. are you want it ?

**Comment by mazharul** [comment]
Revert back index.html file. Add your changes to cv.html file and push your commit. I will review the changes then.

**Comment by mazharul** [comment]
Also - Don't forget to add the redirect rules. I have given you the documentation above.

File: index.html
Comment (mazharul) [inline index.html:1]: Revert the file back.

**Review by mazharul — CHANGES_REQUESTED** [review]
Revert the index file back
Add redirect rule for netlify. See comments for documentation 
Make the page responsive.

**Review by mazharul — CHANGES_REQUESTED** [review]
Revert the index file back
Add redirect rule for netlify. See comments for documentation 
Make the page responsive.

**Comment by Ziaurrahaman7** [comment]
Redirect file added

**Comment by mazharul** [comment]
Your build is failing - fix the redirect rule.

**Review by mazharul — CHANGES_REQUESTED** [review]
index.html file is still not reverted. Change back the index.html file - discard your changes to that file. 
Fix the redirect rule.
I can only validate your responsive page once the above are fixed.

**Review by mazharul — CHANGES_REQUESTED** [review]
Hey - not sure if you aren't getting it or if I am conveying it wrongly. 

- Revert back the content of index.html file to the original content.

**Comment by Ziaurrahaman7** [comment]
i did not found any revert option and it does not  merged

File: cv.html [hunk 1]
```diff
@@ -0,0 +1,381 @@
...
+        <li>Managing deliveries of multiple teams to continuously build features and add values to ookyo oDerings</li>
+        <li>Work with the product to define the roadmap, translating this into user journeys and technical stories for the teams to work on</li>
+        <li>Ensuring product quality and improving ookyo development process</li>
+        <li>Worked with technical leads of the teams to design the systems for scalability and maintainability</li>
+      </ul>
+    </div>
+   </section>
+   <section class="pb-10">
+    <div class="grid afterSection">
+      <h1 class="text-3xl">WORK EXPERIENCE</h1>
```
Comment (mazharul) [inline cv.html]: Remove this duplicated work experience :)

File: cv.html [hunk 2]
```diff
@@ -0,0 +1,381 @@
...
+        <li>Working with regional and international TLD providers; Implementing API wrappers to solve the manual work of buying SEA
+          regional domains. Saving hours of manual work and enabled an automated process to do this. </li>
+        <li>Architecting, developing of the system for high availability and secure DNS management which scaled to handle thousands of
+          domains</li>
+      </ul>
+    </div>
+   </section>
+   <section class="pb-10">
+    <div class="grid afterSection">
+      <h1 class="text-3xl">WORK EXPERIENCE</h1>
```
Comment (mazharul) [inline cv.html]: This is also duplicated.

**Review by mazharul — CHANGES_REQUESTED** [review]

**Review by mazharul — APPROVED** [review]

**Pull request merged**